src/RoomsSDKAdapter.integration.test.js

```diff
@@ -1,42 +1,44 @@
 import {createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
-import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+import WebexSDKAdapter from './';
 
 describe('Rooms SDK Adapter', () => {
-  let roomsSDKAdapter, subscription, user;
+  let createdRoom, getRoom$, webexSDKAdapter, subscription, user;
 
   // Since these are integration tests with live data,
   // increase the async "idle" timeout so jest doesn't error early.
   jest.setTimeout(30000);
 
   beforeAll(async () => {
     user = await createIntegrationTestUser();
-  });
-
-  beforeEach(() => {
-    roomsSDKAdapter = new RoomsSDKAdapter(user.sdk);
+    webexSDKAdapter = new WebexSDKAdapter(user.sdk);
+    await webexSDKAdapter.connect();
   });
 
   afterAll(async () => {
-    // This is done after unsubscribing, but since unsubscribe can't be async, manually call it.
-    await user.sdk.rooms.stopListening();
-
     try {
       await removeIntegrationTestUser(user);
+      await webexSDKAdapter.disconnect();
     } catch (reason) {
       // eslint-disable-next-line no-console
       console.warn('Failed to delete test user for Room SDK Adapter integration tests', reason);
     }
   });
 
   describe('getRoom() returns', () => {
+    beforeEach(async () => {
+      createdRoom = await user.sdk.rooms.create({title: 'Webex Teams Test Room'});
```

Comment:
I don't think it's necessary to create a new room per test, I think one room for this suite should be sufficient

Reply:
The rooms must be created for each test, since we are modifying the data in one of the test cases. However, I did make sure to remove the rooms after being tested.